netlify/functions/mcp.ts

```diff
@@ -10,6 +10,11 @@ import { registerMetadataTools } from "../../tools/metadata.js";
 import { registerPrompts } from "../../prompts/prompts.js";
 
 export default async (req: Request) => {
+  const url = new URL(req.url);
```

Comment:
When hitting the endpoint manually from a netlify preview deploy, it does not respond as expected

Reply:
You didn't change anything